pkg/kv/iterators_test.go

```diff
@@ -86,17 +97,42 @@ func TestIterators(t *testing.T) {
 		if itr.Err() != nil {
 			t.Fatalf("unexpected error: %v", itr.Err())
 		}
+		itr.Close()
 
 		if diffs := deep.Equal(names, []string{"b", "c", "d"}); diffs != nil {
 			t.Fatalf("got wrong list of names: %v", diffs)
 		}
 	})
 
+	t.Run("partition iterator failed SeekGE", func(t *testing.T) {
```

Comment:
The fail here is - not found - the suggested test was related to the internal error as the ScanGE in some of implementations can fail. But not with mem driver.